samples/listFilesPaginate.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function main(bucketName = 'my-bucket') {
+  // [START storage_list_files]
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  async function listFilesPaginated() {
+    const bucket = storage.bucket(bucketName);
+    const [files, queryForPage2] = await bucket.getFiles({autoPaginate: false});
+    console.log('Files:');
```

Comment:
```suggestion

    console.log('Files:');
```

Reply:
I was just suggesting we add a new line before the `console.log`, which it looks like that sample does as well. Sorry for the confusion.

Reply:
Added a line

Reply:
Sorry, this new line should go above the `console.log`:

```js
const [files, queryForPage2] = await bucket.getFiles({autoPaginate: false});

console.log('Files:');
files.forEach(file => {
  console.log(file.name);
});
```

Reply:
✔️ 
Thanks!